_includes/filter-controls.html

```diff
@@ -0,0 +1,94 @@
+<div id="filter-controls">
+  <fieldset>
+    <legend>Filter this page</legend>
+    <div class="filter-options">
+      
+      <fieldset class="filter-group">
+        <legend class="filter-group-label">Accessibility requirements:</legend>
+        <div class="checkbox-group">
+          <div class="checkbox-option">
+            <input type="checkbox" id="level-a" name="level" value="A" checked>
+            <label for="level-a">WCAG 2 Level A</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="level-aa" name="level" value="AA" checked>
+            <label for="level-aa">WCAG 2 Level AA</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="level-aaa" name="level" value="AAA">
+            <label for="level-aaa">WCAG 2 Level AAA</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-aria" name="level" value="aria" checked>
+            <label for="show-aria">WAI-ARIA</label>
+          </div>
+        </div>
+      </fieldset>
+      
+      <fieldset class="filter-group">
+        <legend class="filter-group-label">
+          Rule Status:
+          <a href="#about-rule-status" aria-label="about rule status" class="question">?</a>
```

Comment:
Maybe it's just me but it's confusing visually that it's a link (I expect a button popup). If it has to be a link I would prefer to just have it as text or not have it at all since it already exists in the page contents but it's up to you

Reply:
I'll look at adding a popover to show the text on hover/focus